src/app/payment-hub/transactions/outgoing/outgoing-transactions.component.ts

```
@@ -241,28 +269,58 @@ export class OutgoingTransactionsComponent implements OnInit, AfterViewInit {
+    }
+
+    // Track pagination if paginator exists
+    if (this.paginator) {
+      this.onPageChange(this.paginator.pageIndex, this.paginator.pageSize);
+    }
+
+    // Track performance of transaction loading
+    const startTime = performance.now();
+
     if (!this.sort.direction) {
       delete this.sort.active;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)